openverse_catalog/dags/common/requester.py

```diff
@@ -90,10 +93,34 @@ def get(self, url, params=None, **kwargs):
         except Exception as e:
             logger.error(f"Error with the request for URL: {url}")
             logger.info(f"{type(e).__name__}: {e}")
-            logger.info(f"Using query parameters {params}")
+            if params := request_kwargs.get("params"):
+                logger.info(f"Using query parameters {params}")
             logger.info(f'Using headers {request_kwargs.get("headers")}')
             return None
 
+    def get(self, url, params=None, **kwargs):
+        """
+        Make a GET request, and return the response object if it exists.
+
+        Required Arguments:
+
+        url:      URL to make the request as a string.
+        params:   Dictionary of query string params.
+        **kwargs: Optional arguments that will be passed to `requests.get`.
+        """
+        return self._make_request(self.session.get, url, params=params, **kwargs)
+
+    def head(self, url, **kwargs):
```

Comment:
Just wanted to add the note that the only reason `params` weren't included in the HEAD request was to follow the [Requests API](https://requests.readthedocs.io/en/latest/api/#requests.head).

However I'll be glad to update either method to make them uniform. 👍

Reply:
I think this as good as is! Thanks for the extra note 🙂